modules/mapper-extras/src/main/java/org/opensearch/index/mapper/RankFeaturesFieldMapper.java

```diff
@@ -66,16 +76,20 @@ public Builder(String name) {
 
         @Override
         protected List<Parameter<?>> getParameters() {
-            return Collections.singletonList(meta);
+            ArrayList<Parameter<?>> result = new ArrayList<>(2);
+            result.add(positiveScoreImpact);
+            result.add(meta);
+            return result;
         }
 
         @Override
         public RankFeaturesFieldMapper build(BuilderContext context) {
             return new RankFeaturesFieldMapper(
                 name,
-                new RankFeaturesFieldType(buildFullName(context), meta.getValue()),
+                new RankFeaturesFieldType(buildFullName(context), meta.getValue(), positiveScoreImpact.getValue()),
```

Comment:
I see it being used by `final Parameter<Boolean> positiveScoreImpact`, the `RankFeaturesFieldType` could drop `positiveScoreImpactё` and use it from `RankFeaturesFieldType` since it has a direct reference to it (sorry, copied from https://github.com/opensearch-project/OpenSearch/pull/2726#issuecomment-1087702435)

Reply:
@reta I just follow same style like it's done in:
* `org.opensearch.index.mapper.RankFeatureFieldMapper`
* `org.opensearch.index.mapper.ScaledFloatFieldMapper`

They also have it in `Builder`s.

Also you can check that there `meta` on the same level and it's have same situation:
```
        private final Parameter<Boolean> positiveScoreImpact = Parameter.boolParam(
            "positive_score_impact",
            false,
            m -> ft(m).positiveScoreImpact,
            true
        );
        private final Parameter<Map<String, String>> meta = Parameter.metaParam();
```

I think it's better to be in the same style for now and if needed do refactoring for all places in separate PR.

However, I'm ok if maintainers of this repository do changes that is needed before merging this PR.

Reply:
👍 no objection from be, just feels a bit redundant